doc/classes/BaseMaterial3D.xml

```diff
@@ -390,7 +390,7 @@
 			Repeat flags for the texture. See [enum TextureFilter] for options.
 		</member>
 		<member name="transparency" type="int" setter="set_transparency" getter="get_transparency" enum="BaseMaterial3D.Transparency" default="0">
-			If [code]true[/code], transparency is enabled on the body. Some transparency modes will disable shadow casting. Any transparency mode other than Disabled has a greater performance impact compared to opaque rendering. See also [member blend_mode].
+			The material's transparency mode. Some transparency modes will disable shadow casting. Any transparency mode other than [code]Disabled[/code] has a greater performance impact compared to opaque rendering. See [enum Transparency]. See also [member blend_mode].
```

Comment:
```suggestion
			The material's transparency mode. Some transparency modes will disable shadow casting. Any transparency mode other than [constant TRANSPARENCY_DISABLED] has a greater performance impact compared to opaque rendering. See also [member blend_mode].
```

I don't think we need to link to the enum, it should already be linked in the method signature. We should link to the constant though.